docs/install/index.md

```diff
@@ -158,4 +159,11 @@ macOS:
 make release-macos
 {% endhighlight %}
 
+Windows:
+
+{% highlight bash %}
+cargo build
+cargo run
```

Comment:
Maybe we can say that an executable will be built.

so something like:

```
{% highlight bash %}
cargo build --release
{% endhighlight %}

After the command execution an executable will be created called Rio.exe inside of “target/release”

Optionally you can also build and run the terminal with “cargo run”.
```

Reply:
I found one faster. But you're right I'll leave it as you put it. Thank you